### ISSUE TITLE
'Accessibility mode on'  not persisted across VSCode sessions

### ISSUE THREAD

**Issue by isidorn** [Microsoft/vscode#2304]
* Turn on 'Use tab key for setting focus'
* Restart vscode
* This setting is not persisted, so a user that uses accessibility would need to turn this on every startup

**Comment by bpasero** [comment]
I think that is desired behavior @alexandrudima ?

**Comment by alexandrudima** [comment]
@isidorn `Ctrl+M` is there to get you temporarily out of the editor because the editor eats `Tab` and `Shift+Tab`.

There is no spec I followed when adding this, so I'm not sure what the desired behaviour should be. Maybe you can find an accessibility guide somewhere that mentions how this should work?

**Comment by isidorn** [comment]
@chrisdias please jump in for the accessibility spec

**Comment by chrisdias** [comment]
I don't think this is so much of an accessibility spec question but a usage question.  If I turn on accessibility mode because that is the mode in which i will use VS Code, I expect that mode to persist between sessions. It is similar to picking a theme that I like to work in, I expect that theme to be persisted between sessions, and it is. So yes, this should be persistent.

**Comment by alexandrudima** [comment]
There is a confusion regarding the naming -- the thing is not accessibility mode -- the toggle is that Tab allows focusing out of the editor -- I will soon push a change to the name in the status bar

(It is a similar concept to toggling overwrite with <kbd>Insert</kbd> -- which we don't currently support)

**Comment by alexandrudima** [comment]
Just delivered a new label for the `tabFocusMode`: `"Tab moves focus"`. I hope this clears things up what this feature is: a toggle for inserting a tab character or moving focus when pressing <kbd>Tab</kbd>

![image](https://cloud.githubusercontent.com/assets/5047891/12642701/c03eae70-c5b7-11e5-9181-455b545966c2.png)
